### PR TITLE
Styling the providers list widget

### PR DESCRIPTION
**Pull request by mjchristy**
### Description

Adding styling to the auth account widget. 

image
image

### Scenarios Tested

No existing accounts
Some existing accounts
New account creation
Auto generate user

### CODE REVIEW

File: src/emulator/auth/widget_ui.ts [hunk 1]
```diff
@@ -294,21 +327,67 @@ const STYLE = `
...
-  width: 500px;
+  width: 515px;
+  margin: 16px auto;
 }
 
-button {
-  text-transform: none !important;
+.content-wrapper {
+  padding: 0 24px;
+}
```
Comment (NothingEverHappens) [inline src/emulator/auth/widget_ui.ts:346]: nit: combine with below?

File: src/emulator/auth/widget_ui.ts [hunk 2]
```diff
@@ -294,21 +327,67 @@ const STYLE = `
...
+.mdc-list--avatar-list .mdc-list-item__graphic {
+  border-radius: 50%;
+  height: 36px;
+  margin-left: 0;
+  margin-right: 16px;
+  width: 36px;
+}
+
+.mdc-list .mdc-list-item__graphic {
+  display: -ms-inline-flexbox;
```
Comment (NothingEverHappens) [inline src/emulator/auth/widget_ui.ts:361]: I don't know if we really need all this -ms prefixed stuff, do we even support ie? (I don't think we do in the console)

File: src/emulator/auth/widget_ui.ts [hunk 3]
```diff
@@ -294,21 +327,67 @@ const STYLE = `
...
+}
+
+.mdc-list--avatar-list .mdc-list-item__graphic {
+  background-color: rgba(0,0,0,.3);
+  color: #fff;
+}
+
+.mdc-list-item__graphic {
+  align-items: center;
+  color: background-color: rgba(0,0,0,.3);
```
Comment (NothingEverHappens) [inline src/emulator/auth/widget_ui.ts:372]: nit/optional: consider using a var?

**Review by NothingEverHappens — APPROVED** [review]

File: src/emulator/auth/widget_ui.ts (hunk 1 above)
Comment (mjchristy) [reply src/emulator/auth/widget_ui.ts:346]: Done.

File: src/emulator/auth/widget_ui.ts (hunk 2 above)
Comment (mjchristy) [reply src/emulator/auth/widget_ui.ts]: Ah, copied some of this from material website. I'll drop it.

File: src/emulator/auth/widget_ui.ts (hunk 3 above)
Comment (mjchristy) [reply src/emulator/auth/widget_ui.ts]: We're not really using variables, but I'm gonna go with an opaque gray.

**Comment by yuchenshi** [comment]
Great work!

**Pull request merged**